applications: Asset Tracker_v2: Split `prj.conf` into multiple overlays (#5662)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

* Updated the application to start sending neighbor cell measurement data to nRF Cloud.
* Added content-type and encoding properties to outgoing Azure MQTT messages.
* Added support for A-GPS and P-GPS in Azure IoT Hub integration.

> * Added new overlay configuration files and split the :file:`prj.conf` file into the various files that now bind specific features.

Reviewer: ```suggestion
* Added new overlay configuration files and split the :file:`prj.conf` file to several files that now bind specific features.
```

* :ref:`lib_nrf_cloud` library:

>   * Removed the `CONFIG_NRF_CLOUD` Kconfig option.

Reviewer: ```suggestion
  * Removed the ``CONFIG_NRF_CLOUD`` Kconfig option.
```
